device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/DeviceCalendarPlugin.kt

```diff
@@ -53,7 +62,7 @@ class DeviceCalendarPlugin() : MethodCallHandler {
 
     companion object {
         @JvmStatic
-        fun registerWith(registrar: Registrar): Unit {
+        fun registerWith(registrar: Registrar) {
             val context: Context = registrar.context()
             val activity: Activity? = registrar.activity()
 
```

Comment:
Is there a reason for the creation of two MethodChannel

Reply:
there should've only been one. removed now